.goreleaser.yaml

```diff
@@ -26,14 +26,22 @@ builds:
     flags:
       - -tags=nonsm
 archives:
-  - name_template: "{{ .Binary }}_{{ .Os }}_{{ .Arch }}"
+  - name_template: >-
+      {{ .Binary }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64"}}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
```

Comment:
Does the download script need to be updated to support these changes?

Reply:
I think this will result in the same names we had before based on [this](https://goreleaser.com/deprecations/#archivesreplacements)